src/patcher.js

```diff
@@ -69,7 +71,8 @@ class PatchedBrowserWindow extends BrowserWindow {
     }
 
     opts.webPreferences.enableRemoteModule = true;
-    return new BrowserWindow(opts);
+    const win = new BrowserWindow(opts);
```

Comment:
Any specific reason for assigning a variable and returning it instead of straight up returning the BrowserWindow?

Reply:
Actually none, I forgot to revert that while I was figuring out how to get the original preload file loaded.